source/ceylon/buffer/codec/IncrementalCodec.ceylon

```diff
@@ -23,8 +22,10 @@ shared class ChunkConvert<ToMutable, FromImmutable, ToSingle, FromSingle>(conver
     shared Boolean done => remainder.empty;
```

Comment:
I suggest we remove `done`.

The `PieceConvert` may have additional (partial) input to convert.  But, the `PieceConverter`'s status can
only be checked by calling the similarly named `PieceConverter.done()` method which has the effect of reporting errors and resetting the converter (i.e. it means "Finish now.", not "Are we done?")